src/components/chart/example-clustered-column-chart.njk

```diff
@@ -0,0 +1,73 @@
+{% from "components/chart/_macro.njk" import onsChart %}
+
+{{
+    onsChart({
+        "chartType": "column",
+        "description": "Volume sales, monthly percentage change, seasonally adjusted, Great Britain, October 2024",
+        "theme": "alternate",
+        "title": "Clothing stores fell back, following three months of growth",
+        "subtitle": "Volume sales, monthly percentage change, seasonally adjusted, Great Britain, October 2024",
+        "id": "uuid",
+        "caption": "Source: Monthly Business Survey, Retail Sales Inquiry from the Office for National Statistics",
+        "download": {
+            'title': 'Download Figure 1 data',
+            'itemsList': [
+                {
+                    "text": "Excel spreadsheet (XLSX format, 18KB)",
+                    "url": "#"
+                },
+                {
+                    "text": "Simple text file (CSV format, 25KB)",
+                    "url": "#"
+                },
+                {
+                    "text": "Image (PNG format, 25KB)",
+                    "url": "#"
+                }
+            ]
+        },
+        "legend": {
+                "enabled": true
+            },
+        "series": [
+            {
+                "data": [6.2, 1.5, 15.9, 1.7],
+                "dataLabels": false,
+                "name": "2022"
+            },
+            {
+                "data": [2.9, -2.7, 2.9, 3.5],
+                "dataLabels": false,
+                "name": "2023"
+            },
+            {
+                "data": [-1.6, 1.2, 3.2, 3.8],
+                "dataLabels": false,
+                "name": "2024"
+            },
+            {
+                "data": [3.0, 2, 5.6, 2.1],
+                "dataLabels": false,
+                "name": "2025"
+            },
+            {
+                "data": [2.0, 2, 3, 4],
+                "dataLabels": false,
+                "name": "2026"
+            }
+        ],
+        "xAxis": {
+            "categories": [
+                "Food stores",
+                "Non-food stores",
+                "Non-store retailing ",
+                "Automotive fuel"
+            ],
+            "title": "",
```

Comment:
Theres a few places where we are setting this to "" don't we want to just not set it?

Reply:
We need to explicitly set it to "" to avoid Highcharts adding a default title of 'Values' to an axis